lib/rs/Cargo.toml

```diff
@@ -2,8 +2,11 @@
 
 name = "thrift"
 version = "0.0.1"
-authors = ["Simon Génier <s@simon.coffee>", "Maxim Golov <maxim.golov@gmail.com>"]
+authors = ["Jonathan Reem <jonathan.reem@gmail.com>",
+           "Simon Génier <s@simon.coffee>",
+           "Maxim Golov <maxim.golov@gmail.com>"]
```

Comment:
Yup feel free to edit this however you want I didn't give it basically any thought.